enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Filter/FilterPicker/FilterPicker.tsx

```diff
@@ -32,7 +37,7 @@ const FilterPickerInner = ({
     onQueryChange,
   });
   return (
-    <Box className={className}>
+    <Box className={className} style={style}>
```

Comment:
Nice catch. Did you add `style` (and maybe `className` too?) to all components that are shown in the API doc?

Reply:
No, only to a few components.
If we want it I can do it as M2 task